pom.xml

```diff
@@ -267,7 +267,12 @@
                 <groupId>org.eluder.coveralls</groupId>
                 <artifactId>coveralls-maven-plugin</artifactId>
                 <!-- <version>${coveralls-maven-plugin.version}</version> -->
-            </plugin>
+		</plugin>
+		<plugin>
```

Comment:
We do not plan to use the maven-site-plugin, but instead, rely on GitHub pages. This plugin should be removed.

Reply:
I added it only to see some maven reports and it is executed only if you call it explicitly, so it should be not be in conflict with the GitHub pages. The default version had some issues to be executed, so I had to specify a previous one.
If you want I will remove it anyway.

Reply:
Yes please remove it. You can also call it directly in the command line with a specific version if you need.

Reply:
Ok, removed.